nav.css

```diff
@@ -104,18 +104,42 @@
 	padding-left: var(--ncn-pad-h);
 	padding-right: var(--ncn-pad-h);
 }
+:where(.ncn-home-logo-full) {
+	--_ncn-home-logo-lines-color: var(--ncn-home-logo-lines-color, #05bdba);
+	--_ncn-home-logo-text-color: var(--ncn-home-logo-text-color, #014847);
+}
+.ncn-home-logo-full[data-theme="dark"] {
+	--_ncn-home-logo-lines-color: #32E6E2;
+	--_ncn-home-logo-text-color: #ffffff;
+}
+:where(.ncn-home-logo-encapsulated) {
+	--_ncn-home-logo-lines-color: var(--ncn-home-logo-lines-color, #32E6E2);
+	--_ncn-home-logo-text-color: var(--ncn-home-logo-text-color, #ffffff);
+	--_netlify-logo-fill: var(--netlify-logo-fill, #014847);
+}
+.ncn-home-logo-encapsulated {
+	display: none;
+}
```

Comment:
What's the reason for separating these two rulesets? I know wrapping it in `:where` removes specificity so it's `0,0,0` while the other would be `0,1,0`